.github/workflows/ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: CI
+on: [pull_request, push]
+jobs:
+  #################### Linters and checkers ####################
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v2.3.4
+      - name: Set up Python ${{ matrix.python-version }}
+        id: python
+        uses: actions/setup-python@v2.2.1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: pip install --upgrade pip setuptools wheel tox tox-docker
+      - name: Run flake8
+        run: tox -v -e py-flake8 -- -v
+      - name: Run pydocstyle
+        run: tox -v -e py-pydocstyle -- -v
+      - name: Run apicheck
+        run: tox -v -e py-apicheck -- -v
+  #################### Unittests ####################
+  unittest:
+    needs: lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6,3.7,3.8,pypy3]
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v2.3.4
+      - name: Set up Python ${{ matrix.python-version }}
+        id: python
+        uses: actions/setup-python@v2.2.1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: pip install --upgrade pip setuptools wheel tox tox-docker
+      - name: Run unittest
+        run: tox -v -e ${{ matrix.python-version }}-unit -- -v
```

Comment:
when I first tried using tox integration tests in celery took so long time that I had to try other options. good to see it working good

Reply:
Yes GitHub actions are pretty fast.